assets/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    plugins: [
+        require('precss'),
+        require('autoprefixer')
+    ]
+}
```

Comment:
Maybe 2 spaces indent here just like in webpack config

Reply:
Updated! Thanks 👍